sql/moz-fx-data-marketing-prod/ga_derived/mobile_installs_adjust_v1/query.sql

```diff
@@ -0,0 +1,63 @@
+SELECT
+  date,
+  app,
+  network,
+  campaign,
+  adgroup,
+  creative,
+  CASE
+  WHEN
+    country = 'us'
+  THEN
+    'United States'
+  WHEN
+    country = 'de'
+  THEN
+    'Germany'
+  WHEN
+    country = 'it'
+  THEN
+    'Italy'
+  WHEN
+    country = 'uk'
+  THEN
+    'United Kingdom'
+  WHEN
+    country = 'ca'
+  THEN
+    'Canada'
+  WHEN
+    country = 'es'
+  THEN
+    'Spain'
+  WHEN
+    country = 'pl'
+  THEN
+    'Poland'
+  WHEN
+    country = 'fr'
+  THEN
+    'France'
+  ELSE
+    'Other'
+  END
+  AS country,
```

Comment:
I think I'd just uppercase the country to make it compatible with our country table and handle the aggregation into "Other" in Looker

Reply:
Vanessa's suggestion is reasonable, though for any use cases outside of Looker it'd be less convenient to have to do the country-code-to-name join manually.

If you do want to have the country name in this table I'd suggest joining to `moz-fx-data-shared-prod.static.country_codes_v1` in this query to get it for more than these few hard-coded cases.